zanata-war/src/main/resources/messages.properties

```diff
@@ -1111,6 +1111,7 @@ jsf.language.members.empty=No members
 jsf.language.member.joined=You are a member of this language
 jsf.language.requests=Requests
 jsf.language.myRequests=My Requests
+jsf.language.delete.title=Delete language
 jsf.language.delete.description=Are you sure you wish to delete language this from Zanata?
```

Comment:
This should be fixed as well - "this language" (not language this). Better if the language name could be a part of the string.